modules/nf-core/openms/idmassaccuracy/main.nf

```diff
@@ -0,0 +1,52 @@
+process OPENMS_IDMASSACCURACY {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/openms:3.1.0--h191ead1_4' :
+        'biocontainers/openms:3.1.0--h191ead1_4' }"
+
+    input:
+    tuple val(meta), path(mzmls), path(idxmls)
+
+    output:
+    tuple val(meta), path("*frag_mass_err.tsv") , emit: frag_err
+    tuple val(meta), path("*prec_mass_err.tsv") , emit: prec_err, optional: true
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    IDMassAccuracy \\
+        -in $mzmls \\
+        -id_in $idxmls \\
+        -out_fragment ${prefix}_frag_mass_err.tsv \\
+        -threads $task.cpus \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        OpenMS: \$(FileInfo 2>&1 | grep -E '^Version(.*)' | sed 's/Version: //g' | cut -d ' ' -f 1 | cut -d '-' -f 1)
```

Comment:
This could probably be simpler, what does the initial string look like?

Reply:
Unfortunately, OpenMS doesn't provide me with a version flag. This is the initial string:

```
(csvkit) (base) ubuntu@jonas-ubuntu:/mnt/volume$ FileInfo 2>&1

FileInfo -- Shows basic information about the file, such as data ranges and file type.
Full documentation: http://www.openms.de/doxygen/nightly/html/TOPP_FileInfo.html
Version: 3.2.0-pre-develop-2024-09-06 Sep 10 2024, 21:19:45, Revision: f57a3b9
To cite OpenMS:
 + Pfeuffer, J., Bielow, C., Wein, S. et al.. OpenMS 3 enables reproducible analysis of large-scale mass spectrometry data. Nat Methods (2024). doi:
   10.1038/s41592-024-02197-7.

Usage:
  FileInfo <options>

Options (mandatory options marked with '*'):
  -in <file>*        Input file (valid formats: 'mzData', 'mzXML', 'mzML', 'sqMass', 'dta', 'dta2d', 'mgf', 'featureXML', 'consensusXML', 'idXML', 
                     'pepXML', 'fid', 'mzid', 'trafoXML', 'fasta', 'pqp')
  -in_type <type>    Input file type -- default: determined from file extension or content (valid: 'mzData', 'mzXML', 'mzML', 'sqMass', 'dta', 'dta2
                     d', 'mgf', 'featureXML', 'consensusXML', 'idXML', 'pepXML', 'fid', 'mzid', 'trafoXML', 'fasta', 'pqp')
  -out <file>        Optional output file. If left out, the output is written to the command line. (valid formats: 'txt')
  -m                 Show meta information about the whole experiment
  -p                 Shows data processing information
  -s                 Computes a five-number statistics of intensities, qualities, and widths
  -d                 Show detailed listing of all spectra and chromatograms (peak files only)
  -c                 Check for corrupt data in the file (peak files only)
  -v                 Validate the file only (for mzML, mzData, mzXML, featureXML, idXML, consensusXML, pepXML)
  -i                 Check whether a given mzML file contains valid indices (conforming to the indexedmzML standard)
                     
Common TOPP options:
  -ini <file>        Use the given TOPP INI file
  -threads <n>       Sets the number of threads allowed to be used by the TOPP tool (default: '1')
  -write_ini <file>  Writes the default configuration file
  --help             Shows options
  --helphelp         Shows all options (including advanced)

No options given. Aborting!
```